humanoid_league_team_communication/scripts/show_team_comm.py

```diff
@@ -32,16 +34,14 @@
         SIDE: UNDEFINED
 """
 
-
-class TeamCommPrinter:
-
+class TeamCommPrinter(Node):
     def __init__(self):
-        rclpy.init(args=None)
+        super().__init__("team_comm")
```

Comment:
Same here: should the node have its own name e.g. `team_comm_test`?

Reply:
Yes, but not the same as above

Reply:
But this should be different to the team com test name, Maybe something like `team_comm_viz`?